chapters/annotations.tex

```diff
@@ -2324,6 +2324,19 @@ \subsection{Licensing}\label{licensing}
 \end{example}
 
 
+\section{License Texts}\label{license-texts}
+
+For a top-level class the {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives a license text file for the class.
+The annotation may give a scalar value or an array of values.
+The license text files are human-readable files containing the license conditions.
+If the annotation specifies an array of license text files the conditions in all of them must be satisfied (i.e., it is not intended for dual-licensing).
+External functions may also contain this annotation (with similar content), see \cref{annotations-for-external-libraries-and-include-files}.
+
+When generating a distributable version for a translated model the license text files for used functions and top-level classes should be included.
+Exceptions could be that licenses will be distributed in another way, or that there is an agreement to include different licenses (e.g., for internal libraries).
+Note that the collection of license text files is only intended as a help, thus there is no need for a strict definition of distributable version or stricter rules for including the license text files.
```

Comment:
```suggestion
Note that the \lstinline!License!-annotation only allows the library developer and tool to assist the user in doing what is legally required; in the end it is a user responsibility to ensure that required license files accompany distributable versions of models.
```
As suggested.

Reply:
@henrikt-ma your comment above with changes.

Reply:
@henrikt-ma are you ok with change based on your comment?

Reply:
Yes, it is in the same spirit.